readme.md

```diff
@@ -1,14 +1,14 @@
 # Example Application 
-Exploring howto use Java ~~14~~/~~15~~/16 Records as DTOs in Spring RestController. 
+Exploring howto use Java ~~14~~/~~15~~/~~16~~/17 Records as DTOs in Spring RestController. 
 
 ## Build and Test
 ```
 mvn clean package
 export JAVA_TOOL_OPTIONS=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:8000
-java --enable-preview -jar ./target/playground.java14.records.jar
+java --enable-preview -jar ./target/playground.java17.records.jar
```

Comment:
is `--enable-preview` still required?

Reply:
You are right, not anymore. Will update the pull request